File: Service_eHospital_DongNai_A/Implement/ChungTu_UNCService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class ChungTu_UNCService : BaseService<ChungTu_UNC, IChungTu_UNCRepository>
    , IChungTu_UNCService
    {
    private readonly IChungTu_UNCRepository _ChungTu_UNCRepository;
    public ChungTu_UNCService(IChungTu_UNCRepository ChungTu_UNCRepository) : base(ChungTu_UNCRepository)
    {
    _ChungTu_UNCRepository = ChungTu_UNCRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/ChungTu_UNC.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ChungTu_UNC } from './ChungTu_UNC.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ChungTu_UNCService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'UNC_Id', 'SoUNC', 'NgayUNC', 'GiaTriThanhToan', 'NhaCungCap_Id', 'GhiChu', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: ChungTu_UNC[] | undefined;
    ListFilter: ChungTu_UNC[] | undefined;
    FormData!: ChungTu_UNC;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ChungTu_UNC";
    }
}

